### ISSUE TITLE
OGC document review: Vertical alignment in tables

### ISSUE THREAD

**Issue by manuel489** [metanorma/mn-native-pdf#308]
*In relation to https://github.com/metanorma/mn-native-pdf/issues/89*

In OGC documents, vertical alignment in tables is set to the top by default instead of to center:

*Generated PDF:*
![issue2-pdf_generated](https://user-images.githubusercontent.com/33627611/106648192-55791400-6566-11eb-94da-af8790755215.PNG)

*From template:*
![issue2-pdf_sample](https://user-images.githubusercontent.com/33627611/106648206-58740480-6566-11eb-9258-cc5e1333e4e9.PNG)

**Comment by Intelligent2013** [comment]
@manuel489  in xslt default (if there isn't `valign` attribute) vertical aligment for table cells sets to center.

But in the source XML, each `td` and `th` cell has the attribute `valign="top"`:
```xml
<table id="_a7e8921f-2964-4efb-8143-adf5add2c94d" unnumbered="true">
<thead>
<tr>
<th valign="top" align="left">Name</th>
<th valign="top" align="left">Affiliation</th>
</tr>
</thead>
<tbody>
<tr>
<td valign="top" align="left">Clemens Portele <em>(editor)</em></td>
<td valign="top" align="left">interactive instruments GmbH</td>
</tr>
<tr>
<td valign="top" align="left">Panagiotis (Peter) A. Vretanos <em>(editor)</em></td>
<td valign="top" align="left">CubeWerx Inc.</td>
</tr>
<tr>
<td valign="top" align="left">Charles Heazel <em>(editor)</em></td>
<td valign="top" align="left">Heazeltech LLC</td>
</tr>
<tr>
<td valign="top" align="left">Michael Gordon</td>
<td valign="top" align="left">Ordnance Survey</td>
</tr>
<tr>
<td valign="top" align="left">Jeff Harrison</td>
<td valign="top" align="left">US Army Geospatial Center (AGC)</td>
</tr>
<tr>
<td valign="top" align="left">Chris Holmes</td>
<td valign="top" align="left">Planet Labs</td>
</tr>
<tr>
<td valign="top" align="left">Frédéric Houbie</td>
<td valign="top" align="left">Hexagon</td>
</tr>
</tbody>
</table>
```

Without `valign="top"` (removed manual) PDF displays so:
![изображение](https://user-images.githubusercontent.com/5559926/106714327-bfe48f80-660c-11eb-8672-8aefe5134292.png)

@opoudjis why `valign` default value has been set to `top`? Is it possible to redefine this behaviour?

**Comment by opoudjis** [comment]
valign set to top is the Asciidoc default. valign can now be set explicitly for table cells: https://docs.asciidoctor.org/asciidoc/latest/tables/align-by-cell/#vertical-operators . But in introducing that, I am parsing the Asciidoc value of valign for each cell, and Asciidoc does not allow me to tell the difference between an valign explicitly set to top with `.<`, and a valign defaulting to top.

So in short... no, I can't override that default: if cells are to be valigned to centre, that needs to be given explicitly in the Asciidoc source. If we can be confident that OGC will not accept valign to top, you can override the valigns your side; but I don't know if we can. @ronaldtse, should we ask them?

**Comment by ronaldtse** [comment]
This is poor practice in the parser... Do you think we can try detecting the default and "unset" them if they were not specified by the user? Stylistic concerns should be squarely done on the presentation side.

**Comment by opoudjis** [comment]
As I've said, no I cannot, short of parsing the entire table asciidoc from scratch: the alignment attribute of individual cells is not exposed to the API, only its contents.

**Comment by ronaldtse** [comment]
Question asked at https://github.com/metanorma/metanorma-ogc/issues/250

**Comment by opoudjis** [comment]
Issue has been resolved there.
